src/Widgets/ColumnContainerWidget/ColumnsEditorTab.tsx

```diff
@@ -524,25 +524,23 @@ function adjustNumberOfColumns(
   containerWidget.update({ columns: newColumns })
 }
 
+/**
+ * Copy first n - 1 columns and merge last columns into one
+ */
 function distributeContents(columns: Widget[], originalContents: Widget[][]) {
-  const splitIndexAt = columns.length - 1
-
-  // copy first n -1 elements
-  take(originalContents, splitIndexAt).forEach((originalContent, index) => {
-    columns[index].update({ content: originalContent })
-  })
-
-  // merge last columns into one
-  const colsToMerge = takeRight(
-    originalContents,
-    originalContents.length - splitIndexAt,
+  columns.forEach((column, index) =>
+    column.update({
+      content:
+        index !== columns.length - 1
+          ? originalContents[index]
+          : originalContents.slice(index).flat(),
```

Comment:
I suggest to change the condition to `===`:

```suggestion
        index === columns.length - 1
          ? originalContents.slice(index).flat()
          : originalContents[index],
```

Reply:
My intention was to match the logical order of operations:
1. Fitting columns are copied 1:1 first (top line)
2. The remaining columns are merged and copied to the last index (bottom line)

Does this make sense?